cg/store/models.py

```diff
@@ -884,9 +895,31 @@ class Order(Model):
     id = Column(types.Integer, primary_key=True, unique=True)
     customer_id = Column(ForeignKey("customer.id"), nullable=False)
     customer = orm.relationship(Customer, foreign_keys=[customer_id])
+    links = orm.relationship("OrderCase", back_populates="order")
     order_date = Column(types.DateTime, nullable=False, default=dt.datetime.now())
     ticket_id = Column(types.Integer, nullable=False, unique=True, index=True)
     workflow = Column(types.Enum(*tuple(Workflow)), nullable=False)
 
+    @property
+    def cases(self) -> list[Case]:
+        """Return case samples."""
+        return self._get_cases
+
+    @property
+    def _get_cases(self) -> list[Case]:
+        """Extract samples from a case."""
+        return [link.case for link in self.links]
+
     def to_dict(self):
         return to_dict(model_instance=self)
+
+
+class OrderCase(Model):
+    __tablename__ = "order_case"
+    __table_args__ = (UniqueConstraint("order_id", "case_id", name="_order_case_uc"),)
+
+    id = Column(types.Integer, primary_key=True, unique=True, nullable=False)
+    order_id = Column(ForeignKey("order.id", ondelete="CASCADE"), nullable=False)
+    case_id = Column(ForeignKey("case.id", ondelete="CASCADE"), nullable=False)
+    case = orm.relationship(Case, back_populates="order_links")
+    order = orm.relationship(Order, back_populates="links")
```

Comment:
Not sure what the preferred way is. But for pure link-tables you can define them as line 34-51. Then you can navigate directly between the two models without going via the link model

Reply:
Nice, I will change it!

Reply:
Changed!